requirements.txt

```diff
@@ -1,6 +1,10 @@
-python_version>=3.11
+#python_version>=3.11
```

Comment:
@WebSpider This is one more point worth of discussion.

I have commented this req because main system (ENV) python version can be different than the one used from e.g. Visual Studio Code (where I can pick many installations of Python).

Hence, although I have installed multiple installations (including 3.18, without making it primary), I couldn't pass the check.

What do you suggest here?

Reply:
Currently HA depends on 3.12, so if possible, use 3.12